test/metabase/lib/metadata/jvm_test.clj

```diff
@@ -83,3 +83,34 @@
                 :display-name "Sum"
                 :source_alias "Orders"}]
               (lib.metadata.calculation/metadata mlv2-query))))))
+
+(deftest ^:parallel temporal-bucketing-options-test
+  (mt/dataset sample-dataset
+    (let [query {:lib/type :mbql/query
+                 :stages   [{:lib/type     :mbql.stage/mbql
+                             :fields       [[:field
+                                             {:lib/uuid (str (random-uuid))}
+                                             (mt/id :products :created_at)]]
+                             :source-table (mt/id :products)}]
+                 :database (mt/id)}
```

Comment:
Aren't we trying to avoid writing pMBQL queries by hand, but rather using the helpers from `metabase.lib.dev` etc.?

Reply:
Will fix in a different PR.